test/development/acceptance-app/hydration-error.test.ts

```diff
@@ -47,6 +47,25 @@ describe('Error overlay for hydration errors', () => {
       See more info here: https://nextjs.org/docs/messages/react-hydration-error"
     `)
 
+    await session.patch(
+      'app/page.js',
+      outdent`
+      'use client'
+      const isClient = typeof window !== 'undefined'
+      export default function Mismatch() {
+        return (
+          <div className="parent">
+            <main className="child">Value</main>
```

Comment:
Should it use `isClient` and generate some different content on server?

Currently I don't see how it is a mismatch?

Reply:
This is the recovery case, the mismatch is introduced here: https://github.com/vercel/next.js/pull/60423/files#diff-80445b3939cc69811f786ad7ee91e9be2a5553cf67e5b4d175519204bcd7c64dR31